scripts/apps/authoring/authoring/services/AuthoringService.js

```diff
@@ -50,8 +50,9 @@ export function AuthoringService($q, $location, api, lock, autosave, confirm, pr
      * @param {string} _id Item _id.
      * @param {boolean} readOnly
      * @param {string} repo - repository where an item whose identifier is _id can be found.
+     * @param {string} action - action performed to open the story: edit, correct or kill
      */
-    this.open = function openAuthoring(_id, readOnly, repo) {
+    this.open = function openAuthoring(_id, readOnly, repo, action) {
```

Comment:
I think it should be ok. ArchiveLock endpoint handles the null value for action and saves as 'edit' if it is null.